### PR TITLE
COMMERCE-9373

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-9373

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 1]
```diff
@@ -197,6 +197,138 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-9373 Verify pagination is working properly in preview when importing a Order from Placed Orders"
+	@priority = 5
+	test CanChangePaginationInPreviewWhenImportOrderFromPlacedOrders {
+		property portal.acceptance = "true";
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:203]:
```suggestion
	@priority = 3
	test CanUseImportFromOrdersPreviewPagination {
		property portal.acceptance = "false";
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 2]
```diff
@@ -197,6 +197,138 @@ definition {
...
+
+		task ("Given the Money Order payment is activated") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			CommerceNavigator.gotoPortlet(
+				category = "Store Management",
+				portlet = "Channels");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:212]:
```suggestion
			CommerceChannels.openChannelsAdmin();
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 3]
```diff
@@ -197,6 +197,138 @@ definition {
...
+				totalPrice = "100.00",
+				unitPrice = "100.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Wear Sensors",
+				productQuantity = 1,
+				productSku = "MIN93027",
+				totalPrice = "60.00",
+				unitPrice = "60.00");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:276]: @Brunno0liveira I don't think we need to assert all these details for this test. I believe the sku/name is enough.
This applies to all these assertions in the 3 tests

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 4]
```diff
@@ -197,6 +197,138 @@ definition {
...
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Wear Sensors",
+				key_value = "MIN93027",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			IFrame.closeFrame();
+		}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:329]: The assertion of the previous page arrow is missing. Also paginating by directly clicking on a page number is missing

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase [hunk 5]
```diff
@@ -32,6 +32,111 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "COMMERCE-9373 Verify pagination is working properly in preview when importing a CSV Order File"
+	@priority = 5
+	test CanChangePaginationInPreviewWhenImportCSVOrderFile {
+		property portal.acceptance = "true";
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase:38]:
```suggestion
	@description = "COMMERCE-9373 Verify pagination is working properly in preview when importing a CSV"
	@priority = 3
	test CanUseImportFromCSVsPreviewPagination {
		property portal.acceptance = "false";
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase [hunk 6]
```diff
@@ -32,6 +32,111 @@ definition {
...
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Master Cylinder",
+				key_value = "MIN93026",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			IFrame.closeFrame();
+		}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase:137]: The assertion of the previous page arrow is missing. Also paginating by directly clicking on a page number is missing

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceWishList.testcase [hunk 7]
```diff
@@ -591,6 +591,131 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-9373 Verify pagination is working properly in preview when importing a Wish List"
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceWishList.testcase:594]:
```suggestion
	@description = "COMMERCE-9373 Verify pagination is working properly in preview when importing a wish list"
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceWishList.testcase [hunk 8]
```diff
@@ -591,6 +591,131 @@ definition {
...
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			AssertElementNotPresent(
+				key_columnNumber = 1,
+				key_productName = "Premium Brake Pads",
+				key_value = "MIN93019",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			IFrame.closeFrame();
+		}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceWishList.testcase:716]: The assertion of the previous page arrow is missing. Also paginating by directly clicking on a page number is missing

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceWishList.testcase [hunk 9]
```diff
@@ -591,6 +591,131 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-9373 Verify pagination is working properly in preview when importing a Wish List"
+	@priority = 3
+	test CanChangePaginationInPreviewWhenImportOrderFromWishList {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceWishList.testcase:596]:
```suggestion
	test CanUseImportFromWishListsPreviewPagination {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceWishList.testcase [hunk 10]
```diff
@@ -591,6 +591,131 @@ definition {
...
+		task ("Given a Minium site with a buyer user") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And given some products in wish list") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceWishList.testcase:608]:
```suggestion
		task ("And a wish list with some products") {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 11]
```diff
@@ -250,6 +250,128 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-9373 Verify pagination is working properly in preview when importing an order"
+	@priority = 5
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:254]: Please adjust priority according to my previous comment

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase (hunk 5 above)
Comment (lorenzolapescara) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase:38]: @Brunno0liveira this is missing

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase [hunk 12]
```diff
@@ -32,6 +32,103 @@ definition {
...
+				key_columnNumber = 1,
+				key_productName = "Master Cylinder",
+				key_value = "MIN93026",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			Click(locator1 = "ACUtils#PREVIOUS_PAGES");
+
+			Click(
+				key_pageNumber = 3,
+				locator1 = "ACUtils#ANY_PAGE");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase:116]:
```suggestion
			Click(locator1 = "ACUtils#PREVIOUS_PAGES");

			CommerceEntry.viewPaginationResults(results = "Showing 1 to 20 of 216");

			CommerceOrders.assertOrderImportPreviewEntry(
				productName = "Brake Fluid",
				productSku = "MIN93016A");

			Click(
				key_pageNumber = 3,
				locator1 = "ACUtils#ANY_PAGE");
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase [hunk 13]
```diff
@@ -250,6 +250,128 @@ definition {
...
+				key_columnNumber = 1,
+				key_productName = "Wear Sensors",
+				key_value = "MIN93027",
+				locator1 = "CommerceEntry#TABLE_SEARCH_BY_NAME_AND_COLUMN_CONTENT");
+
+			Click(locator1 = "ACUtils#PREVIOUS_PAGES");
+
+			Click(
+				key_pageNumber = 3,
+				locator1 = "ACUtils#ANY_PAGE");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase:359]:
```suggestion
			Click(locator1 = "ACUtils#PREVIOUS_PAGES");

			CommerceEntry.viewPaginationResults(results = "Showing 1 to 4 of 13");

			CommerceOrders.assertOrderImportPreviewEntry(
				productName = "ABS Sensor",
				productSku = "MIN93015");

			Click(
				key_pageNumber = 3,
				locator1 = "ACUtils#ANY_PAGE");
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]
@Brunno0liveira please check the comments. Thanks!

**Review by lorenzolapescara — APPROVED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Pull request closed without merging**